api/internal/handlers/linux/install.go

```diff
@@ -403,3 +415,44 @@ func (h *Handler) GetAppInstallStatus(w http.ResponseWriter, r *http.Request) {
 
 	utils.JSON(w, r, http.StatusOK, appInstall, h.logger)
 }
+
+// calculateRegistrySettings calculates registry settings for airgap installations
+func (h *Handler) calculateRegistrySettings(ctx context.Context) (*types.RegistrySettings, error) {
```

Comment:
the expectation is that the handlers package should be a light wrapper to the business logic. this logic has high potential of being re-used somewhere else as well. how about moving this to the linux install manager and expose it via the linux install controller and then use it here?